src/main/java/com/desiato/puresynth/controllers/AudioController.java

```diff
@@ -19,24 +16,23 @@ public final class AudioController {
     private final AudioService audioService;
 
     @PostMapping("/generate")
-    public ResponseEntity<?> generateAudio(@RequestBody final AudioRequest request) {
+    public ResponseEntity<String> generateAudio(@RequestBody final AudioRequest request) {
```

Comment:
What does this string mean as a return type?

Reply:
I wanted to be more specific, but is it better to leave `?` in case the return type changes?

Reply:
It's even worse. You should return a structured response. Returning `File generated: " + audioFile.fileName()` is very bad. What the user has to do with this? You're creating a classic REST API. All your other endpoints return JSON objects, why something like that here?